File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-1.cpp
class Solution {
public:
    int evalRPN(vector<string>& tokens) {
        stack<int> st;
        for(string c:tokens){
            if(c=="+"||c=="-"||c=="*"||c=="/"){
                int a=st.top();
                st.pop();
                int b=st.top();
                st.pop();
                switch(c[0]){
                case '+':
                    st.push(a+b);
                    break;
                case '-':
                    st.push(b-a);
                    break;
                case '*':
                    st.push(a*b);
                    break;
                case '/':
                    st.push(b/a);
                    break;
                }
            }else{
                int x=stoi(c);
                st.push(x);
            }
        }
        return st.top();
    }
};

File: Data Structures & Algorithms/n-ary-tree-postorder-traversal/submission-1.java
class Solution{
    public List<Integer> postorder(Node root){
        Stack<Node> st=new Stack<>();
        List<Integer> result=new ArrayList<>();
        if(root!=null) st.push(root);
        while(!st.isEmpty()){
            Node node=st.pop();
            result.add(node.val);
            for(Node child:node.children) st.push(child);
        }
        Collections.reverse(result);
        return result;
    }
}